PowerNSX.psm1

```diff
@@ -19847,7 +19847,10 @@ function New-NsxSecurityTag {
             [string]$Name,
         [Parameter (Mandatory=$false)]
             [string]$Description,
+        [Parameter (Mandatory=$false)]
+            [switch]$Universal=$true,
         [Parameter (Mandatory=$False)]
+
```

Comment:
Im terrible at this, but if you can try to avoid random blank lines.  Also - if you dont mind - can you add comments for each param (Im trying to do this each time I make changes to a cmdlet that doesnt already have them)